src/pudl/extract/excel.py

```diff
@@ -267,7 +267,10 @@ def extract(self, **partitions):
                 # check if there are any missing or extra columns
                 str_part = str(list(partition.values())[0])
                 col_map = self.METADATA._column_map[page]
-                page_cols = col_map.loc[col_map[str_part].notnull(), [str_part]].index
+                page_cols = col_map.loc[
+                    (col_map[str_part].notnull()) & (col_map[str_part] != -1),
+                    [str_part],
+                ].index
```

Comment:
Before I added this, you would get a ton of warnings every time you tried to extract 861 data because of all the rows with -1 that don't show up every year. `page_cols` was just filtering for NA. By adding the -1 filter, those warnings go away, and the expected_cols match what's actually there.